zookeeper/Dockerfile

```diff
@@ -49,14 +71,56 @@ COPY shared/log4shell.sh /bin
 RUN /bin/log4shell.sh /stackable/apache-zookeeper-${PRODUCT}-bin
 
 # Ensure no vulnerable files are left over
-# This will currently report vulnerable files being present, as it also alerts on
-# SocketNode.class, which we do not remove with our scripts.
-# Further investigation will be needed whether this should also be removed.
+# This will currently report vulnerable files being present, as it also alerts
+# on SocketNode.class, which we do not remove with our scripts. Further
+# investigation will be needed whether this should also be removed.
 COPY shared/log4shell_1.6.1-log4shell_Linux_x86_64 /bin/log4shell_scanner_x86_64
 COPY shared/log4shell_1.6.1-log4shell_Linux_aarch64 /bin/log4shell_scanner_aarch64
 COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/apache-zookeeper-${PRODUCT}-bin
 # ===
 
+# END ZOOKEEPER BUILDER
+# ==============================================================================
+
+# ==============================================================================
+# START FINAL ZOOKEEPER IMAGE
+
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache ZooKeeper" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache ZooKeeper." \
+      description="This image is deployed by the Stackable Operator for Apache ZooKeeper."
+
+RUN microdnf update && \
+    microdnf install tar gzip zip && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+# COPY --chown=stackable:stackable zookeeper/stackable /stackable
+COPY zookeeper/licenses /licenses
+
+USER stackable
+WORKDIR /stackable
+
+# Copy over the complete Zookeeper binary folder
+COPY --link --chown=stackable:stackable --from=builder /stackable/apache-zookeeper-${PRODUCT}-bin /stackable/apache-zookeeper-${PRODUCT}-bin/
```

Comment:
@Maleware didn't we have problems with `--link`? I used it as well but I remember that we had to remove it?

Reply:
We had, but it worked out. During this I builded zookeeper 3.6.1 and worked on my machine. So fine from my side.

Reply:
I encounter this error: https://github.com/docker/buildx/issues/1408

Reply:
removed the links